packages/react-component-library/src/components/SelectBase/SelectBaseOption.tsx

```diff
@@ -6,28 +6,17 @@ import { StyledOptionBadge } from './partials/StyledOptionBadge'
 import { StyledOptionText } from './partials/StyledOptionText'
 import { ComponentWithClass } from '../../common/ComponentWithClass'
 
-export interface SelectBaseOptionBaseProps extends ComponentWithClass {
+export interface SelectBaseOptionProps extends ComponentWithClass {
   badge?: string | number
   icon?: React.ReactNode
   isHighlighted?: boolean
   value: string
   title?: string
 }
-
-export interface SelectBaseOptionAsArrayProps
-  extends SelectBaseOptionBaseProps {
-  children: React.ReactNodeArray
-}
-
-export interface SelectBaseOptionAsStringProps
-  extends SelectBaseOptionBaseProps {
+export interface SelectBaseOptionAsStringProps extends SelectBaseOptionProps {
```

Comment:
Seems like the blank line between the interfaces has been lost here – would be good to add it back in